### PR TITLE
Update community-plugin-marketplace.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Replaced content with content from https://github.com/mattermost/mattermost-developer-documentation/pull/747.

Content changes are from a while back but were not merged. They include more details around the type of documentation required, as well as an update on the language/terminology. Linked to https://github.com/mattermost/mattermost-marketplace/pull/210 but not dependent on it.

### CODE REVIEW

File: site/content/integrate/plugins/community-plugin-marketplace.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
-6. The plugin has to be out of Beta and be released with at least v1.0.0.
-7. All configuration is accessible via the Mattermost interface.
-8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
+4. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
+5. The plugin has to be out of Beta and be released with at least v1.0.0.
+6. All configuration is accessible via the Mattermost interface.
+7. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```
Comment (hanzei) [inline site/content/integrate/plugins/community-plugin-marketplace.md:23]: Nit why: Why was the hugo ref removed?

File: site/content/integrate/plugins/community-plugin-marketplace.md [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
 
-### Technical Requirements (Checked by developers of the Toolkit or the Integrations team)
+**Technical requirements (checked by developers of the Toolkit team or the Integrations team)**
```
Comment (hanzei) [inline site/content/integrate/plugins/community-plugin-marketplace.md:25]: Given that these two team don't exist any longer `Interstellar` should be used here.

File: site/content/integrate/plugins/community-plugin-marketplace.md [hunk 3]
```diff
@@ -68,24 +68,35 @@ This is checked in dev review.
...
+If a community plugin doesn’t make it through the review process, it may still be added to the Marketplace and marked as “Beta”. The reviewers decide whether the quality of a plugin is sophisticated enough to be added to the Marketplace on a case-by-case basis. Security and functional reviews and items 1, 2, 3, and 5 from the [Product Requirements Checklist](#requirements-for-adding-community-plugin-to-the-marketplace) must be fulfilled for Beta plugins.
 
-It must be made clear in the Plugin Marketplace UI that a plugin is in Beta. Beta plugins must only be visible on Mattermost servers that support labels (i.e. Mattermost v5.20 or later).
+It must be made clear in the Marketplace interface that a plugin is in Beta. Beta plugins must only be visible on Mattermost servers that support labels (i.e. Mattermost v5.20 or later).
```
Comment (hanzei) [inline site/content/integrate/plugins/community-plugin-marketplace.md:94]: I think we can drop this now. The marketplace does sophisticated filtering now.

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks Justine!

File: site/content/integrate/plugins/community-plugin-marketplace.md (hunk 1 above)
Comment (justinegeffen) [inline site/content/integrate/plugins/community-plugin-marketplace.md:23]:
```suggestion
7. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
```

Comment (justinegeffen) [reply site/content/integrate/plugins/community-plugin-marketplace.md:23]: Not sure actually, but it's back in!

File: site/content/integrate/plugins/community-plugin-marketplace.md (hunk 2 above)
Comment (justinegeffen) [inline site/content/integrate/plugins/community-plugin-marketplace.md:25]:
```suggestion
**Technical requirements (checked by developers of the Interstellar team)**
```

File: site/content/integrate/plugins/community-plugin-marketplace.md [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
-7. All configuration is accessible via the Mattermost interface.
-8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
+4. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
+5. The plugin has to be out of Beta and be released with at least v1.0.0.
+6. All configuration is accessible via the Mattermost interface.
+7. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
 
-### Technical Requirements (Checked by developers of the Toolkit or the Integrations team)
+**Technical requirements (checked by developers of the Toolkit team or the Integrations team)**
```
Comment (justinegeffen) [inline site/content/integrate/plugins/community-plugin-marketplace.md]:
```suggestion
**Technical requirements (checked by developers of the Interstellar team)**
```

**Review by hanzei — APPROVED** [review]
Nice :+1:

File: site/content/integrate/plugins/community-plugin-marketplace.md [hunk 5]
```diff
@@ -1,63 +1,63 @@
...
-7. All configuration is accessible via the Mattermost interface.
-8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
+4. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
+5. The plugin has to be out of Beta and be released with at least v1.0.0.
+6. All configuration is accessible via the Mattermost interface.
+7. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
 
-### Technical Requirements (Checked by developers of the Toolkit or the Integrations team)
+**Technical requirements (checked by developers of the Interstellar team)**
```
Comment (BenLloydPearson) [inline site/content/integrate/plugins/community-plugin-marketplace.md:25]: Will external developers know what "Interstellar team" refers to?

**Review by BenLloydPearson — APPROVED** [review]
I have one comment, but this looks great otherwise.

File: site/content/integrate/plugins/community-plugin-marketplace.md (hunk 5 above)
Comment (justinegeffen) [reply site/content/integrate/plugins/community-plugin-marketplace.md:25]: Good question! We can change this to "Checked by the relevant development team".

Comment (justinegeffen) [inline site/content/integrate/plugins/community-plugin-marketplace.md:25]:
```suggestion
**Technical requirements (checked by the relevant development team)**
```

**Pull request merged**